pulp_ansible/app/migrations/0047_ansiblerepository_last_sync_task.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.16 on 2023-01-23 18:29
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0099_versions_field'),
+        ('ansible', '0046_add_fulltext_search_fix'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='ansiblerepository',
+            name='last_sync_task',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='ansible_ansiblerepository', to='core.task'),
+        ),
```

Comment:
You are going to need to change the migration file as well to remove `blank`.

Reply:
oops, forget about that, thanks :)